iFixit/react-commerce
# Sentry: Setup in react-commerce

Reports errors to our new [`react-commerce`](https://sentry.io/organizations/ifixit/issues/?project=6469069) project in Sentry in prod and [`react-commerce-dev`](https://sentry.io/organizations/ifixit/issues/?project=6475315) project in dev.

Source maps should work for Next.js (assuming an auth token is available in the environment vars). It's unlikely that they work for the Strapi integration, wasn't sure how to add them there since the JS gets generated by Strapi (I think?).

## QA
Throw some errors!

Make sure that you have no problems/diff when installing from a clean slate, running the project. Verify that errors thrown anywhere in frontend/ are reported to Sentry. I think /backend will report errors when handling requests (not when starting the server fails), but wasn't sure how to trigger one of these errors.

Closes #302
